clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -585,4 +586,20 @@ static CamundaClientCloudBuilderStep1 newCloudClientBuilder() {
    * @return a builder for the process instance query
    */
   ProcessInstanceQuery newProcessInstanceQuery();
+
+  /**
+   * Executes a search request to query user tasks.
+   *
+   * <pre>
+   * zeebeClient
+   *  .newUserTaskInstanceQuery()
```

Comment:
🔧 We should also update the Javadoc accordingly.
```suggestion
   *  .newUserTaskQuery()
```